core/arch/arm/mm/core_mmu.c

```diff
@@ -837,12 +843,14 @@ static void init_mem_map(struct tee_mmap_region *memory_map, size_t num_elems)
 	assert(va >= CFG_TEE_RAM_START);
 	assert(end <= CFG_TEE_RAM_START + CFG_TEE_RAM_VA_SIZE);
 
-	add_pager_vaspace(memory_map, num_elems, va, end, &last);
+	add_pager_vaspace(memory_map, num_elems, va, end, &last,
+			  &next_pgdir_va);
 
```

Comment:
The reason I don't want to enforce pgdir alignment of `va` is primary for LPAE. It seems wasteful to forbid usage of otherwise unused entries.